readme.md

```diff
@@ -82,6 +89,9 @@ var Panda = function(name, age) {
 
 Your Answer:
 ```text
+Panda.prototype.eat_bamboo = function(num_bamboo_eaten + 1) {
+    this.eat_bamboo = num_bamboo_eaten;  
+}
 
 ```
```

Comment:
Very close. You just need to clean up your variable and make sure you're using the right ones in the right place. Should look something like this...

```js
Panda.prototype.eat_bamboo = function() {
  this.num_bamboo_eaten = this.num_bamboo_eaten + 1;
}
```